blog.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name= "viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
```

Comment:
You'll generally want to put utility css above the custom css you write. So ti doesn't overwrite what you wrote. Order matters with a lot of code